src/main/scala/com/crealytics/spark/excel/DataColumn.scala

```diff
@@ -30,6 +30,12 @@ class HeaderDataColumn(
     }
 
     lazy val dataFormatter = new DataFormatter()
+    // Overwrite ExcelGeneralNumberFormat with custom PlainNumberFormat.
+    // See https://github.com/crealytics/spark-excel/issues/321
+    lazy val plainNumberFormat = PlainNumberFormat.getInstance()
+    dataFormatter.addFormat("General", plainNumberFormat)
+    dataFormatter.addFormat("@", plainNumberFormat)
```

Comment:
Do we want to add a config to enable / disable this behaviour (similar to `treatEmptyValuesAsNulls` in this class?

Reply:
Depending on whether it is possible that this breaks somebody's workflow, it might be a good idea :wink: